### PR TITLE
Adds EMP Defensive Traps to Clockcult 

### PR DESCRIPTION
**Pull request by BurgerLUA**
[Changelogs]:
:cl: BurgerBB
add: Adds a few important clockcult tips.
add: Added Mech Sensors, a brass-created trap that activates when a mech not controlled by a cultist crosses it.
add: Added power nullifiers, an emp trap that emps everything in a 3x3 area, with the center suffering a heavy EMP.
tweak: Brass Skewers now deal 50 damage to mechs.
 
/:cl:

[why]: Basically if you have a mech, even if it's a ripley, you can steamroll clock cult very easily as walls are meaningless to you, and the traps do not affect you at all. This PR changes that.

During testing, a gygax without any emp resistance would have to cross 4 brass spear + emp trap combos to be destroyed. This is quite reasonable, as EMP traps are quite visible.

### CODE REVIEW

**Comment by Poojawa** [comment]
why not just make mechs unable to be walked through portals if you're going to be this heavy handed with your unbalancing.

**Comment by BurgerLUA** [comment]
uhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh because it's balanced

**Comment by Trilbyspaceclone** [comment]
>Emps

File: code/modules/antagonists/clockcult/clock_structures/traps/power_null.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	icon_state = "electric_trap"
+	break_message = "<span class='warning'>The power nullifier sparks, then slowly crumbles to debris!</span>"
+	max_integrity = 40
+	density = FALSE
+	var/activated = FALSE
+
+/obj/structure/destructible/clockwork/trap/power_nullifier/activate()
+	if(!activated)
+		activated = TRUE
+		empulse(get_turf(src),1,1,TRUE)
```
Comment (Poojawa) [inline code/modules/antagonists/clockcult/clock_structures/traps/power_null.dm:14]: "it's balanced" he says, knowing full well people will spam these to make energy weapons completely useless.

Comment (BurgerLUA) [reply code/modules/antagonists/clockcult/clock_structures/traps/power_null.dm:14]: Well maybe they treat EMP traps like brass spears and destroy them before they get close.

It's not like the EMP traps have legs. Basically if get crew to destroy the traps before they blindly walk in with energy weapons, then they can be destroyed.

I mean you get the exact same result if you walk into a brass skewer carrying an energy weapon: You drop it and usually someone with a high IQ steals it. I've done this plenty of times as a drone. With this in mind, it's actually worse to emp energy weapons as both sides can't use it.

**Review by silicons — APPROVED** [review]
I don't see an issue with this. Clockcult's in a bad spot right now, this'll hopefully make it better.
If it's too OP we can rebalance.

**Pull request merged**